pytorch_lightning/plugins/training_type/sharded.py

```diff
@@ -41,6 +41,7 @@ def configure_ddp(self) -> None:
             sharded_optimizer=self.lightning_module.trainer.optimizers,
             # For multi-node training, enabling bucketing will improve performance.
             reduce_buffer_size=self._REDUCE_BUFFER_SIZE_DEFAULT if self.num_nodes > 1 else 0,
+            **self._ddp_kwargs
```

Comment:
@SeanNaren should we explicitly check if `reduce_buffer_size` is part of the kwargs to avoid errors with it being configured twice?

Reply:
great point, will do a followup here!